jscomp/super_errors/super_typecore.ml

```diff
@@ -26,18 +26,19 @@ let rec bottom_aliases = function
   | _ -> None
 
 let simple_conversions = [
-  (("float", "int"), "int_of_float");
-  (("int", "float"), "float_of_int");
-  (("int", "string"), "string_of_int");
-  (("float", "string"), "string_of_float");
+  (("float", "int"), "Belt.Float.toInt");
+  (("float", "string"), "Belt.Float.toString");
+  (("int", "float"), "Belt.Int.toFloat");
+  (("int", "string"), "Belt.Int.toString");
+  (("string", "float"), "Belt.Float.fromString");
+  (("string", "int"), "Belt.Int.fromString");
```

Comment:
This is the unclear one for me. It returns an option and has a runtime: https://github.com/rescript-lang/rescript-compiler/blob/4f4812aa4e1411029392f6dbcacc580e755d68d5/jscomp/others/belt_Int.ml#L37